### ISSUE TITLE
GoogleAds is downcased to Googleads

### ISSUE THREAD

**Issue by samphippen** [googleapis/gapic-generator-ruby#138]
see: https://gist.github.com/samphippen/c492b3cd3be0fe30481ad022d5792e05 vs https://github.com/googleapis/googleapis/blob/master/google/ads/googleads/v1/services/campaign_service.proto#L33

**Comment by blowmage** [comment]
Yeah, I think this fix will come with #88. We have several libraries where we will want to use custom names, like `BigQuery` and `PubSub`.

**Comment by samphippen** [comment]
@blowmage this did not work when I pulled the new version of the generator locally: 

```
module Google
  module Ads
    module Googleads
      module V1
        module Services
          module CampaignService
```

`master@fb3dfc0`
